app/models/micropost.rb

```diff
@@ -0,0 +1,17 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  validates :user_id, presence: true
```

Comment:
validates này với validates :user thì khác gì nhau?

Reply:
Su khac nhau : (validates :user_id) la validate foreiogin key association giua Micropost va User. Nhung theo em nghi thi neu validate :user cung duoc vi  khi dung cac method user.build, user.create thi` user_id   automatically set to the right user.